### ISSUE TITLE
[BUG] Trakt syncing error

### ISSUE THREAD

**Issue by bills176** [nixgates/plugin.video.seren#566]
Ref previous Bug #415.

**Logs**
https://paste.ubuntu.com/p/8WCCPF2Gw4/

**Describe the bug**
Seren keeps trying to sync "show bookmarks" with Trakt and fails. It will keep trying as long as Kodi is open. Returns "Rebuilding Trakt Database" then "Unknown error connecting to Trakt". The progress bar at the top is always on "Syncing Show Bookmarks:" The progress bar finishes on watched shows, movies etc. I cannot control the show play progress. All other Trakt operation seem to work.

**To Reproduce**
Start Kodi. Seren automatically syncs at Kodi startup.

**Expected behavior**
Sync of bookmarks. 

Screenshots

Kodi Version (please complete the following information):

Kodi Version 18.9 20201023
Seren Version 2.0.17

**Additional context**
Since Bug 415 I submitted have completely rebuilt my Win10 machine. I have a clean install of Win10, Clean install of Kodi 18.9 and a clean install of Seren 2,0,16, just updated to 2.0.17

**Comment by bills176** [comment]
Previous log was at Kodi startup. The following log is after a rebuild database command from tools menu.
https://paste.ubuntu.com/p/KYc5KMxPwH/

**Comment by cgcgcgcgcg** [comment]
I made a new Matrix build and had the same issue.
So, I ran a Total Clean Up from my wizard, I did the 3 steps shown in the pic, I enabled debug logging, and restarted the build to prepare kodi.log.
I played an episode in Seren Collection, I browsed the build, even waited for several minutes, but the popup never appeared.
![ksnip_20210326-024211](https://user-images.githubusercontent.com/38960370/112561156-50557a00-8ddd-11eb-8f17-de72fe9b7911.png)
